.github/workflows/develop_build_status.yml

```diff
@@ -0,0 +1,29 @@
+name: 'develop build status'
```

Comment:
We can remove develop specific collateral.

Reply:
Removed this file